src/fold/fold.rs

```diff
@@ -153,7 +153,7 @@ fn fold_file<T: Read>(mut file: BufReader<T>, bytes: bool, spaces: bool, width:
                                 match rfind_whitespace(slice) {
                                     Some(m) => {
                                         let routput = &slice[m + 1 .. slice.chars().count()];
-                                        let ncount = routput.chars().fold(0usize, |out, ch: char| {
+                                        let ncount = routput.chars().fold(0, |out, ch: char| {
```

Comment:
Not sure if this util should simply be removed. As fas as I can tell it will only ever work with glibc based binaries.